### DISCUSSION TITLE
Extend Configuration to Support All Stoplight Elements Options

### DISCUSSION THREAD

**Discussion by yondifon** [dedoc/scramble#542] [Ideas]
Hey

First, thank you for making this package—it has been incredibly helpful for APIs. 

I would like to request an enhancement to extend the configuration files to accept all options provided by Stoplight Elements, as outlined here: (https://docs.stoplight.io/docs/elements/b074dc47b2826-elements-configuration-options).

Will offer more flexible 

I'm willing to contribute a PR, just let me know if it's acceptable.

**Comment by romalytvynenko** [comment]
Thanks for asking. I recommend simply publishing docs view and passing all the options there

**Reply by yondifon** [reply]
well that works too.
